app/src/main/kotlin/com/simplemobiletools/flashlight/helpers/MyCameraImpl.kt

```diff
@@ -271,19 +279,19 @@ class MyCameraImpl private constructor(val context: Context, private var cameraT
     }
 
     fun getMaximumBrightnessLevel(): Int {
-        return cameraFlash!!.getMaximumBrightnessLevel()
+        return cameraFlash?.getMaximumBrightnessLevel() ?: MIN_BRIGHTNESS_LEVEL
```

Comment:
shouldnt we return MAX?

Reply:
or it would be easier to show some error if something goes wrong, this can lead to unexpected behaviour and we just wont know about it... Im not a fan of hiding issues silently.

Reply:
Should it just be a toast or should we go for something like disabling these features of the app in case we run into this situation?

Reply:
lets show a toast with given nullpointerexception, maybe it will give us some helpful info that someone sends us and we can fix it properly.